core/src/main/resources/hudson/util/HudsonIsRestarting/index_fr.properties

```
@@ -20,5 +20,8 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
+Restarting \Jenkins=Redémarrage de Jenkins
 Please\ wait\ while\ Jenkins\ is\ restarting=Veuillez attendre pendant que Jenkins redémarre
-Your\ browser\ will\ reload\ automatically\ when\ Jenkins\ is\ ready.=Votre navigateur se connectera automatiquement quand Jenkins sera prêt
+Your\ browser\ will\ reload\ automatically\ when\ Jenkins\ is\ ready.=Votre navigateur se rafraîchira automatiquement quand Jenkins sera prêt.
+Safe\ Restart=Redémarrage sécurisé
```

```suggestion
Safe\ Restart=Redémarrage sans danger
```
Or
```suggestion
Safe\ Restart=Redémarrage sûr
```
More adapted, as safe here has no relation to security, the restart isn't secured.

---

Yes, that makes more sense ^^